addons/product/tests/test_product_attribute_value_config.py

```diff
@@ -365,12 +365,13 @@ def test_get_first_possible_combination(self):
         self.assertEqual(self.computer._get_first_possible_combination(), computer_ssd_256 + computer_ram_16 + computer_hdd_2)
 
         # Move down the ram, so it will try to change the ram instead of the hdd
-        # Test invalidate_cache on product.attribute write
-        self.ram_attribute.sequence = 10
+        # Invalidate the cache to force the application of ordering changes.
+        self.computer_ram_attribute_lines.sequence = 15
```

Comment:
Keep the previous test intact (especially if attribute_id is kept in the sequence as suggested), and make a new test for when lines are reordered.